packages/create-bison-app/template/src/env.mjs

```diff
@@ -0,0 +1,96 @@
+import { z } from 'zod';
+
+const nodeEnv = process.env.NODE_ENV || 'development';
+
+/**
+ * Specify your server-side environment variables schema here. This way you can ensure the app isn't
+ * built with invalid env vars.
+ */
+const server = z.object({
```

Comment:
I've found a helpful naming convention when working zod to be including `Schema` at the end of schema variables (e.g. `serverSchema`).